modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectActionLocalServiceTest.java

```diff
@@ -1819,6 +1824,97 @@ public void testDeleteObjectAction() throws Exception {
 		_objectActionLocalService.deleteObjectAction(systemObjectAction);
 	}
 
+	@Test
+	public void testDTOValueInSystemObjectDefinitionActionPayload()
```

Comment:
@JhossephG please rename it to `testExecuteObjectActionWithUnmodifiableSystemObjectDefinition`